src/client/containers/OrderConfirmationPage/OrderConfirmationPage.styles.css

```diff
@@ -1,3 +1,57 @@
-div {
-  color: #687808;
+.confirmation-page {
+    align-items: center;
+    margin: 4%;
 }
+
```

Comment:
In general, try very hard to avoid two things:
1. CSS class names that include the HTML tag name (`.inner-div`, what happens if the HTML changes to be a span?)
2. Means with low context, such as `wrapper`, a wrapper for what? It is usually possible to think of better names that describe the content on the page!

Reply:
Thanks